src/Chirp.Web/Chirp.Web.csproj

```diff
@@ -23,7 +24,13 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="7.0.10" /> 
```

Comment:
We already have `Microsoft.VisualStudio.Web.CodeGeneration.Design version 7.0.11`, so unless the latest version is unstable, I'd recommend removing `7.0.10`, as the potential downgrade is also noted as an error :)

Reply:
Done